lib/puppet_litmus/serverspec.rb

```diff
@@ -102,6 +102,33 @@ def run_shell(command_to_run, opts = {})
     result
   end
 
+  # Copies file to the target, using its respective transport
+  #
+  # @param source [String] place locally, to copy from.
+  # @param destination [String] place on the target, to copy to.
+  # @param opts [Hash] Alters the behaviour of the command. Valid options are :expect_failures [Boolean] doesnt return an exit code of non-zero if the command failed.
+  # @yieldreturn [Block] this method will yield to a block of code passed by the caller; this can be used for additional validation, etc.
+  # @return [Object] A result object from the command.
+  def bolt_upload_file(source, destination, options = {})
+    inventory_hash = inventory_hash_from_inventory_file
```

Comment:
This will fail if TARGET_HOST is 'localhost' at this line https://github.com/puppetlabs/puppet_litmus/blob/master/lib/puppet_litmus/inventory_manipulation.rb#L15

Reply:
hadnt thought about running bolt against localhost